lib/faro.dart

```diff
@@ -116,18 +124,20 @@ class Faro {
         collectorUrl: optionsConfiguration.collectorUrl ?? '',
       );
     }
-    if (Platform.isAndroid || Platform.isIOS) {
-      NativeIntegration.instance.init(
-          memusage: optionsConfiguration.memoryUsageVitals,
-          cpuusage: optionsConfiguration.cpuUsageVitals,
-          anr: optionsConfiguration.anrTracking,
-          refreshrate: optionsConfiguration.refreshRateVitals,
-          setSendUsageInterval: optionsConfiguration.fetchVitalsInterval);
+    if (!_platformInfoProvider.isWeb) {
+      if (Platform.isAndroid || Platform.isIOS) {
```

Comment:
If we now have a `_platformInfoProvider`, why not use that to check isAndroid or isIOS?